tableauserverclient/server/endpoint/datasources_endpoint.py

```diff
@@ -16,7 +17,7 @@
 logger = logging.getLogger('tableau.endpoint.datasources')
 
 
-class Datasources(Endpoint):
+class Datasources(TaggedResourcesEndpoint):
```

Comment:
This design concerns me -- I don't like the idea of adding several types of Endpoints in the inheritance chain. What if we need to change endpoints for some other reason?

We can discuss offline, but maybe there's a compositional approach?